creating-virtual-machines/virtio-win.adoc

```diff
@@ -1,53 +1,186 @@
+There you should see undetected devices in "available devices" section. You can install virtio drivers one by
+one going through this list.
+
+
+For more details on how to choose a proper driver and how to install the driver, 
+please refer to the https://access.redhat.com/articles/2470791[Windows Guest Virtual Machines on Red Hat Enterprise Linux 7].
+
+
+How to obtain virtio drivers? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The virtio Windows drivers are distributed in a form of 
+https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/disks-and-volumes.html#containerDisk[containerDisk],
+which can be simply mounted to the VirtualMachine.
+The container image, containing the disk is located at: https://hub.docker.com/r/kubevirt/virtio-container-disk
+and the image be pulled as any other docker container: 
+
+[bash]
+----
+docker pull kubevirt/virtio-container-disk
+----
+
+However, pulling image manually is not required, it will be downloaded if not present 
+by Kubernetes when deploying VirtualMachine.
+
+
+Attaching to VirtualMachine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 KubeVirt distributes virtio drivers for Microsoft Windows in form if
-container disk. The package contains the virtio drivers as well as qemu
-guest agent. The disk was tested on Microsoft Windows Server 2012.
+container disk. The package contains the virtio drivers 
+The disk was tested on Microsoft Windows Server 2012.
 However, it should work on Microsoft Windows 7 and up.
 
-Package is intended to be used as CD-ROM attached to virtual machine
-with Microsoft Windows. It can be used either during install phase as
-SATA CD-ROM to provider drivers or in existing installation as SATA
-CD-ROM to provide QEMU Guest Agent installation.
+The package is intended to be used as CD-ROM attached to the virtual machine
+with Microsoft Windows. It can be used both during install phase as
+SATA CD-ROM to provide drivers or in an existing installation as SATA
+CD-ROM. 
+
+Attaching the virtio-win package can be done simply by adding ContainerDisk
+to you VirtualMachine.
 
-Example how to attach:
+[source,yaml]
+----
+spec:
+  domain:
+    - disk:
+        <any other disk you want to use, must go before virtio>
+        <or choose boot order explicitly>
+    - cdrom:
+        bus:sata
+      name: <name of your disk>
+      bootOrder: 1 // optional, either sort disks or specify bootOrder
+volumes:
+  - containerDisk:
+      image: kubevirt/virtio-container-disk // upstream virtio-win container disk
+    name: <name of your disk>
+----
+
+Once you are done installing virtio drivers, you can remove virtio container disk
+by simply deleting the disk from yaml specification and restarting the VirtualMachine.
+
+The full example of the VirtualMachine running Windows and virtio drivers as
+SATA CD-ROM follows. Please note, you would need to provide your local
+windows disk, to make this example work.
```

Comment:
if you make the previous example nicer then there's no need for this complete yaml. It will keep changing with the rest of kubevirt and you'd need to keep maintaining it here...doesn't look worth it to me.